components/application-operator/README.md

```diff
@@ -31,6 +31,7 @@ Application Operator (AO) performs different operations as a result of the follo
  - **eventServiceImage** is the Event Service image version to use in the Application chart.
  - **eventServiceTestsImage** is the Event Service Tests image version to use in the Application chart.
  - **applicationConnectivityValidatorImage** is the Application Connectivity Validator image version to use in the Application chart.
+ - **strictMode** is a toggle to enable/disable Istio authorization policy for validator and HTTP source adaptor. The default value is `disabled`.
```

Comment:
We should align on the way we do this. Ping @montaro, this concerns you since you used an env var.

Reply:
Will keep it as is as @montaro's PR is not going in and there is no visible benefit in using env var.

Reply:
`adapter` not `adaptor` ;)  more below :)